operator/pkg/helmreconciler/reconcile.go

```diff
@@ -167,7 +171,7 @@ func (h *HelmReconciler) processRecursive(manifests ChartManifestsMap) *v1alpha1
 			status := v1alpha1.InstallStatus_NONE
 			var err error
 			if len(m) != 0 {
-				if processedObjs, err = h.ProcessManifest(m); err != nil {
+				if processedObjs, err = h.ProcessManifest(m, len(componentDependencies[cn]) > 0); err != nil {
```

Comment:
i wonder if the wait option makes sense anymore since anyway we are waiting for the big components. perhaps it's time to take out the wait flag completely?

Reply:
I am fine either way - maybe as a separate PR?